packages/sdk-browser/src/__tests__/utils-test.ts

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from "@jest/globals";
+import { createEmbeddedAPIKey } from "../utils";
+
+test("createEmbeddedAPIKey", async function () {
+  // Test valid uncompressed public key
+  const uncompPubKey =
+    "0400d2eb47be2006c29db5fe9941dd686d19ddeea85a0328894f08091f6b5be9b366c4872345594c12a7f7c47c62dd8074542934820fce5ee0ddc55d6d1d8dd312";
+  await expect(createEmbeddedAPIKey(uncompPubKey)).resolves.not.toThrow();
```

Comment:
We could do a basic check of the return value: verify that the resulting `authBundle` and `publicKey` are valid, non empty strings